stories/object-status/__snapshots__/object-status.stories.storyshot

```diff
@@ -28,20 +28,87 @@ exports[`Storyshots Components/Object Status Clickable Object Status 1`] = `
   </a>
   
 
-  <span
-    class="fd-object-status fd-object-status--critical fd-object-status--link sap-icon--status-critical"
+  <a
```

Comment:
Is there a reason this can't be a button instead of an `a` with `role="button"`?